mrbgems/mruby-io/src/io.c

```diff
@@ -1258,8 +1258,13 @@ mrb_io_s_select(mrb_state *mrb, mrb_value klass)
 retry:
   n = select(max, rp, wp, ep, tp);
   if (n < 0) {
+#ifdef _WIN32
+    if (WSAGetLastError() != WSAEINTR)
+      mrb_sys_fail(mrb, "select failed");
```

Comment:
I am wondering if `WSAGetLastError()` sets `errno`.
If it does not, it might be preferable to have `errno = EINTR` before `mrb_sys_fail()`.

🐈‍⬛ I may PR it if I can confirm it (but I don't know when that will be).

Reply:
Ah, right. I'll fix in later.